c2corg_api/search/sync.py

```diff
@@ -62,13 +64,27 @@ def remove_doc():
             # https://github.com/c2corg/v6_api/issues/89
             DBSession.refresh(document)
 
+        is_user_profile = isinstance(document, UserProfile)
+        if is_user_profile:
+            # set user login + full-name as document title so that it can
+            # be searched
+            user_login, user_name = DBSession. \
+                query(User.name, User.username). \
+                filter(User.id == document.document_id). \
+                first()
+            user_title = '{0} {1}'.format(user_name or '', user_login or '')
```

Comment:
> Is the pseudo visible in v6?

You mean the forum pseudo? Isn't it shown anyway in the posts?
By the way I guess it is not available in the API tables, only in Discourse database? Thus not indexable by ES?

I suggest we keep the user indexing (User.name, User.username) as is and change later if this is a problem for the client.

By the way, could we use ES indexing of the users to associated users to an outing?